packages/angular_devkit/build_angular/src/webpack/configs/styles.ts

```diff
@@ -222,7 +224,14 @@ export function getStylesConfig(wco: WebpackConfigOptions) {
           extracted,
         }),
         ...extraPostcssPlugins,
-        autoprefixer(),
+        postcssPresetEnv({
+          // tslint:disable-next-line: no-any
+          browsers: supportedBrowsers as any, // Typinmgs only allow a string
+          autoprefixer: {
+            overrideBrowserslist: supportedBrowsers,
```

Comment:
Looks like it will automatically set this if the `browsers` option is used:
https://github.com/csstools/postcss-preset-env/blob/d7652b1e6196e8f55bf3f0aac4ac090fec7ed54e/src/postcss.js#L26